diff --git a/android/build.gradle b/android/build.gradle
index 0000000000000000000000000000000000000000..0000000000000000000000000000000000000000 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -141,7 +141,7 @@ android {
         ndkPath rootProject.ext.ndkPath
     }
     if (rootProject.hasProperty("ndkVersion")) {
-        ndkVersion rootProject.ext.ndkVersion
+        ndkVersion = rootProject.ext.ndkVersion
     }
 
     defaultConfig {
