diff --git a/android/build.gradle b/android/build.gradle
index 0000000000000000000000000000000000000000..0000000000000000000000000000000000000000 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -26,7 +26,7 @@ android {
         targetSdkVersion safeExtGet('targetSdkVersion', 31)
     }
     lintOptions {
-        abortOnError false
+        abortOnError = false
     }
 }
 
@@ -35,11 +35,11 @@ repositories {
     mavenCentral()
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-        url "$rootDir/../node_modules/react-native/android"
+        url = "$rootDir/../node_modules/react-native/android"
     }
     maven {
         // Android JSC is installed from npm
-        url "$rootDir/../node_modules/jsc-android/dist"
+        url = "$rootDir/../node_modules/jsc-android/dist"
     }
     google()
 }
